composer.json

```
@@ -10,28 +10,29 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
         "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
         "symfony/translation": "^2.8 || ^3.3 || ^4.0"
     },
     "require-dev": {
         "symfony-cmf/testing": "^2.1@dev",
-        "symfony/phpunit-bridge": "^3.3 || ^4.0",
+        "symfony/phpunit-bridge": "^5",
         "doctrine/phpcr-odm": "^1.4|^2.0",
         "jms/serializer": "^1.2",
         "behat/behat": "^3.0.6",
-        "behat/web-api-extension": "^1.0@dev",
-        "matthiasnoback/symfony-dependency-injection-test": "^1.1",
-        "matthiasnoback/symfony-config-test": "^2.1",
+        "imbo/behat-api-extension": "^2.1",
+        "matthiasnoback/symfony-dependency-injection-test": "^4",
+        "matthiasnoback/symfony-config-test": "^4",
         "symfony/twig-bundle": "^2.8 || ^3.3 || ^4.0",
         "symfony/validator": "^2.8 || ^3.3 || ^4.0",
         "symfony/security-bundle": "^2.8 || ^3.3 || ^4.0",
         "symfony/asset": "^2.8 || ^3.3 || ^4.0",
         "symfony/templating": "^2.8 || ^3.3 || ^4.0",
         "symfony/form": "^2.8 || ^3.3 || ^4.0",
         "symfony/web-server-bundle": "^2.8 || ^3.3 || ^4.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
+        "symfony/translation": "^2.8 || ^3.3 || ^4.0",
+        "phpspec/prophecy": "~1.0"
```

as this is for dev only, i think there is no cost in forcing to a new version that works for sure. 

```suggestion
        "phpspec/prophecy": "^1.12"
```

---

Ah. I've just thought that is a stable version.

---

it is, but maybe there are some new features that have been added meanwhile. --prefer-lowest applies to everything. for the dev-dependencies, we don't care about supporting a wide range - that is only valuable with the main dependencies, so that the bundle can be installed in as many environments as possible. (within reasons, which is why it was ok to drop php 7.1 and we eventually will want to drop symfony 2.8, likely when adding symfony 5 support)